data/yaml/venues/jlcl.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: JLCL
+name: Issue 1
```

Comment:
Should be "Journal for Language Technology and Computational Linguistics", and needs `type: journal`.